### PR TITLE
[BACKPORT] Update Kafka base to version 1.1.0 (#2509)

### PR DESCRIPTION
**Pull request by samvantran**
This backports PR #2509 which upgrades Kafka to version 1.1.0

### CODE REVIEW

File: frameworks/kafka/tests/client.py [hunk 1]
```diff
@@ -93,7 +93,7 @@ def install(self, kerberos: sdk_auth.KerberosEnvironment=None) -> dict:
             "container": {
                 "type": "MESOS",
                 "docker": {
-                    "image": "elezar/kafka-client:4b9c060",
+                    "image": "samvantran/kafka-client:4.1.0",
```
Comment (samvantran) [inline frameworks/kafka/tests/client.py]: @elezar I made a [PR against your repo](https://github.com/elezar/dcos-clients/pull/1) to update the kafka-client. Feel free to build an updated image and I'd be perfectly okay w/ setting this back to `elezar/kafka-client:xx`

Comment (elezar) [reply frameworks/kafka/tests/client.py]: Thanks @samvantran. I have merged your changes and rebuilt `elezar/kafka-client:deca3d0`. Note that the last part is the SHA of the repo after your merged changes, so let's rather switch to that for now. There is a JIRA ticket out to handle our Docker images more consistently.

**Comment by samvantran** [comment]
👍 sounds good @elezar. 

PR #2542 updates the kafka-client on master. I'll combine that with #2534 once we know that the client is backwards compatible. 

Here's the ticket to capture the error we saw during testing (with resolution): [DCOS-38359](https://jira.mesosphere.com/browse/DCOS-38359)

**Comment by samvantran** [comment]
Yup, I won't merge this PR  until 
1) we update the official Kafka-client for #2542
2) CI tests reliably

**Comment by elezar** [comment]
@samvantran with regards to
> cherry pick merged #2542 into this one

We really should close this PR and then cherry-pick the merge commit from the master branch. It may be better to close this PR and open a new one once the other PR is merged.

**Comment by samvantran** [comment]
Okay closing this PR so that it can be a proper cherry-pick backport PR

**Pull request closed without merging**